code/game/atoms.dm

```diff
@@ -293,6 +301,69 @@ its easier to just keep the beam vertical.
 /atom/proc/fire_act()
 	return
 
+/atom/proc/scp914_act(var/mode) //DON'T OVERRIDE, override scp914_act_on_type instead
+	var/list/chosen_mode = modes_914[mode]
+	if(chosen_mode && chosen_mode.len > 0)
```

Comment:
`length()` returns 0 for null so `length(chosen_mode) > 0` is perfect here.

Reply:
got it